build.gradle

```diff
@@ -854,6 +855,9 @@ project('test:system') {
         compile group: 'org.glassfish.jersey.connectors', name:'jersey-apache-connector', version:jerseyVersion
         compile group: 'com.spotify', name: 'docker-client', version: dockerClientVersion
         compile group: 'io.kubernetes', name: 'client-java', version: k8ClientVersion
+        runtime group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: jerseyVersion
+        runtime group: 'org.apache.curator', name: 'curator-framework', version: apacheCuratorVersion
+        testCompile group: 'org.apache.curator', name: 'curator-framework', version: apacheCuratorVersion
```

Comment:
Why do we need this both for `runtime` and `testCompile`? Shouldn't `compile` be enough?

Reply:
The `runtime` configuration is required to pull in the dependency into the `test-collection.jar` (below). It is however sufficient just to use `compile` instead of `testCompile`.

Reply:
If you use `compile` dependency config, the dependency is available for both `runtime` and `testCompile` (as well as `testRuntime`). However, since `compile` is more permissive than what you may need here, IMO it'd be better to specify `runtime` and `testCompile` (or better still `testRuntime` if that's what you need) to limit the scope of these dependencies. That being said, it's not a big deal using just `compile` too. Just a thought.